views/index.erb

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="app.css" />
+    <meta charset="UTF-8">
+    <title>Spotifyifyly</title>
+  </head>
+  <body TEXT="#FFFFC3" LINK="#00FFAE" VLINK="#00FFAE" ALINK="#FFFF00">
```

Comment:
Can we test this out to see if these work? If so, can we move them to the stylesheet with some updated conventions, my dude?